src/store/metadata/stub/content/editions/versions/total-value-of-uk-exports-2022-08.json

```diff
@@ -0,0 +1,88 @@
+            "identifier": "1",
+            "issued": "2023-05-23T10:14:20",
+            "title": "Total value of UK exports",
+            "summary": "Estimated total value of UK exports including trade in both goods and services (experimental statistics), UK, 2021.",
+            "description": "This dataset shows the estimated total value of UK exports including trade in both goods and services (experimental statistics), in the UK, for 2021.\n\n### Coverage \n\nUK \n\n#### Geography definition\n\nInternational Territorial Level (ITL) 1, 2 and 3,\nCity Regions,\nCountry (UK)\n\n### Source\n\n - UK: [Office for National Statistics](https://www.ons.gov.uk/businessindustryandtrade/internationaltrade/bulletins/internationaltradeinuknationsregionsandcities/previousReleases) \n\n\n",
+            "download_url": "https://staging.idpd.uk/datasets/total-value-of-exports/editions/2022-08/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+                "columns": [
+                    {
+                      "name": "period",
+                      "datatype": "string",
+                      "titles": "Period",
+                      "description": "The time period associated with the observation."
+                    },
+                    {
+                      "name": "areacd",
+                      "datatype": "string",
+                      "titles": "ONS Geography Code",
+                      "description": "The geography code associated with the observation."
+                    },
+                    {
+                      "name": "areanm",
+                      "datatype": "string",
+                      "titles": "Area Name",
+                      "description": "The name assiciated with The geography code "
+                    },
+                    {
+                      "name": "geography",
+                      "datatype": "string",
+                      "titles": "Geography Type",
+                      "description": "The categorisation of the geography code"
+                    },
+                    {
+                      "name": "value",
+                      "datatype": "decimal",
+                      "titles": "Geographic area with Total value of UK exports by at least one provider",
+                      "description": "Observational Value"
+                    },
+                    {
+                      "name": "measure",
+                      "datatype": "string",
+                      "titles": "Measure Type",
+                      "description": "Measure Type of the Observational Value"
+                    },
+                    {
+                      "name": "unit",
+                      "datatype": "string",
+                      "titles": "Unit",
+                      "description": "Unit Type of the Observational Value"
+                    },
+                    {
+                      "name": "lower confidence interval (95%)",
```

Comment:
These names should be consistent with the other recently updated datasets: lower_confidence_interval_95 instead of lower confidence interval (95%)

Reply:
done